config/routes.rb

```diff
@@ -12,6 +12,9 @@
    get 'pages/index' => 'pages#index'
 
    resources :courses
+
+   post 'learning/apply' => 'learning#apply'
+   get 'learning/show' => 'learning#show'
```

Comment:
I don't think that we need show page and show view, because Learning is kinda intermidiate table to display what we need